pedpy/methods/method_utils.py

```diff
@@ -287,11 +290,28 @@ def compute_neighbors(
             needs to contain a column 'polygon', which holds a
             :class:`shapely.Polygon` (result from
             :func:`~method_utils.compute_individual_voronoi_polygons`)
+        as_list (bool): Return the neighbors as a list per pedestrian and frame,
+            if true, otherwise each neighbor is in a single row.
```

Comment:
I added a warning in the notebook that discourages usage of the function with the default value. Do we additionally want to emit a deprecation warning if it is set to True?

Reply:
After thinking about: I also added a deprecation warning, this way everybody gets informed when using the function, without any need to read a up-to-date documentation. This way more user will update their scripts (hopefully).

Reply:
I like this explicit solution better.